less/variables.less

```diff
@@ -11,4 +11,13 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+// Media Queries
+@mobile: ~"(max-width: 500px)";
```

Comment:
Great use of escaping to save yourself the hassle of typing that over and over again and making it easy to change across the whole page if needed.